projects/observability/src/shared/components/utils/chart-tooltip/chart-tooltip-popover.ts

```diff
@@ -11,7 +11,7 @@ export class ChartTooltipPopover<TData, TContext> implements ChartTooltipRef<TDa
   ) {}
 
   public showWithData(element: Element, data: MouseLocationData<TData, TContext>[]): void {
```

Comment:
Since we're not showing the tooltip in the sync'd charts (discussed on slack), no changes should be needed in this file or any of the other popover files.